src/main/java/com/google/codeu/utilities/MessageUtil.java

```diff
@@ -48,4 +65,12 @@ public static String formatImages(String message){
 
     return textWithImages;
   }
+
+  /*
+   * This method takes the message entered by the user and replaces markdown with
+   * html tags in the html of the page using the regular expressions above (regex)
+   */
+  public static String formatText(String message) {
+	  return message.replaceAll(BOLD_REGEX, BOLD_REPLACEMENT).replaceAll(ITALIC_REGEX, ITALIC_REPLACEMENT).replaceAll(STRIKE_REGEX, STRIKE_REPLACEMENT).replaceAll(UNDERLINE_REGEX, UNDERLINE_REPLACEMENT);
```

Comment:
Nitpick: It's a bit easier to read if you put each replaceAll(...) on a separate line. Java knows the line doesn't end until the semicolon, so common practice is to break before the period, for example:

return message.replaceAll(...)
    .replaceAll(...)
    .replaceAll(...);